### PR TITLE
Feature/fiscal–entity

### PR DESCRIPTION
**Pull request by Kentverger**

### CODE REVIEW

File: lib/Conekta/Conekta.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 abstract class Conekta
 {
     public static $apiKey;
-    public static $apiBase = 'https://api.conekta.io';
+    public static $apiBase = 'http://localhost:61440';
```
Comment (andreasantillana) [inline lib/Conekta/Conekta.php]: this should point to production

File: lib/Conekta/Resource.php [hunk 2]
```diff
@@ -137,11 +137,21 @@ protected function _createMember($member, $params)
         $requestor = new Requestor();
         $url = $this->instanceUrl().'/'.$member;
         $response = $requestor->request('post', $url, $params);
-        if (strpos(get_class($this->$member), 'Object') !== false || strpos($member, 'cards') !== false || strpos($member, 'payout_methods') !== false) {
+
+        var_dump($response);
```
Comment (andreasantillana) [inline lib/Conekta/Resource.php]: remove var_dump

File: test/Conekta.php [hunk 3]
```diff
@@ -5,7 +5,7 @@ function setApiKey()
 {
     $apiEnvKey = getenv('CONEKTA_API');
     if (!$apiEnvKey) {
-        $apiEnvKey = '1tv5yJp3xnVZ7eK67m4h';
+        $apiEnvKey = 'WVWIDUUBc4ca4238a0b9';
```
Comment (andreasantillana) [inline test/Conekta.php]: change it back

**Comment by fruizg0302** [comment]
👍

**Pull request merged**